src/tools/papi-connector/README.md

```diff
@@ -1,10 +1,46 @@
-### Build papi connector for kokkos
+## Using the PAPI Connector
 
+To use the PAPI connector you have to compile it against the latest PAPI version that includes the new high-level API.
+
+Details about PAPI's new high-level API can be found here:  
+https://bitbucket.org/icl/papi/wiki/Home
+
+#### Install PAPI (from version 6.0) or latest git version
+
+```console
+git clone https://bitbucket.org/icl/papi.git
+```   
+Installation instructions can be found here:  
+https://icl.utk.edu/papi/software/index.html
+
+#### Install the PAPI connector
+```console
 export PAPI_ROOT=<Path of PAPI>  
-make clean  
-make  
+make
+```
+This will generate a dynamic library `kp\_papi\_connector.so`.
```

Comment:
```suggestion
This will generate a dynamic library `kp_papi_connector.so`.
```

I just realize you are not supposed to escape the underscores for inline code (between backticks)